package.json

```diff
@@ -22,6 +22,7 @@
     "@svgr/webpack": "^5.5.0",
     "clsx": "^1.1.1",
     "file-loader": "^6.2.0",
+    "prism-react-renderer": "^1.2.1",
```

Comment:
yarn.lock?

Reply:
The original code theme can't highlight some key words and can't change along with dark mode.

![after](https://user-images.githubusercontent.com/52615090/128628759-5e8f60d1-a5a4-4997-a654-8c7049f92ae6.jpeg)



![original](https://user-images.githubusercontent.com/52615090/128628788-0a92b428-2822-45e4-bae8-10f2172e02ea.jpeg)

Reply:
The prism-react-renderer package is added by docusaurus in yarn.lock: https://github.com/casdoor/casdoor-website/blob/master/yarn.lock#L6700